src/TenantManager.php

```diff
@@ -67,15 +68,30 @@ public function createTenant(Tenant $tenant): self
         $afterCreating = [];
 
         if ($this->shouldMigrateAfterCreation()) {
-            $afterCreating += $this->databaseCreationQueued() ? [
+            $afterCreating = array_merge($afterCreating, $this->databaseCreationQueued() ? [
```

Comment:
Note: I changed these to `array_merge` because the `+=` was behaving like `array_unique` instead. :(

Reply:
Yeah, I also noticed some weird behavior with that operator. Thought the syntax looked nice. Look at my PR to your `seeder` branch, I use `[]` there, since the arrays have only one item anyway.

Reply:
Oh, duh. Just adding using `$foo[] =` is definitely easier!